module1/src/com/epam/cycles/HomeTask_1_3_7.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.cycles;
+
+import java.util.Scanner;
+
+public class HomeTask_1_3_7 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Введите а: ");
+        int a = in.nextInt();
+        System.out.println("Введите b: ");
+        int b = in.nextInt();
+        int[] arr = new int[10];
+        while(a != 0){
+            arr[a % 10]++;
+            a /=10;
+        }
+        while(b != 0){
+            if(arr[b % 10] != 0)
+                System.out.print(b % 10 + " ");
+            b /=10;
```

Comment:
1)You use b%10 twice, so define a variable for it
2)It is a good practise to use if (...) {} even if you have one string in this block of code
```suggestion
            int digitOfB = b % 10;
            if(arr[digitOfB] != 0) {
                System.out.print(digitOfB + " ");
            }
            b /=10;
```